iele-gas.md

```diff
@@ -828,22 +845,39 @@ Note: These are all functions as the operator `#compute` has already loaded all
       requires L1 >=Int L2
     
     rule Cdiv(SCHED, L1, L2) =>
-        Gdivword < SCHED > *Int L1 +Int //TODO(remove this term)
+        Gdivword < SCHED > *Int L1 +Int
+        Gdiv < SCHED >
+      requires notBool Gnewarith << SCHED >>
+      [owise]
+
+    rule Cdiv(SCHED, L1, L2) =>
         Gdiv < SCHED >
+      requires Gnewarith << SCHED >>
       [owise]
 
     rule Cexp(SCHED, L1, W1, W2) =>
         Gexpkara < SCHED > *Int #overApproxKara(#adjustedBitLength(L1, W1) *Int W2 /Int 64) +Int
-        Gexpword < SCHED > *Int L1 +Int // TODO(adjustedBitLength)
+        Gexpword < SCHED > *Int L1 +Int
```

Comment:
What is the explanation of this new formula?

Reply:
it's the same formula, but we measure L1 at bit-granularity. This gave us a more accurate fit when we were measuring cpu time.

Reply:
That covers `Cexp`, but what about the extra terms for `Cexpmod`.

Reply:
these were also based on improving the fit in our measurements.